k8s/namespaces/wa/wa-task-batch-clean-up/aat-01.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   values:
     job:
-      schedule: "5-59/10 * * * *"
+      schedule: "30 * * * *"
```

Comment:
Will these schedules clash at say 12:00 or 13:00?
Better if we split them?

Reply:
It works like below
next at 2022-08-17 12:30:00
then at 2022-08-17 13:30:00
then at 2022-08-17 14:30:00
then at 2022-08-17 15:30:00
then at 2022-08-17 16:30:00